packages/interface/src/components/complementary-area/index.js

```diff
@@ -150,6 +153,16 @@ function ComplementaryArea( {
 					/>
 				</PinnedItems>
 			) }
+			{ __unstableInitSource !== 'core' && isPinnable && (
+				<ComplementaryAreaMoreMenuItem
+					__unstableInitSource={ __unstableInitSource }
```

Comment:
Feels like we could just drop the `__unstableInitSource` from here.

Reply:
I now connected this comment with https://github.com/WordPress/gutenberg/pull/29081#discussion_r578203796. Yes, that would work 👍🏻